DESCRIPTION

```diff
@@ -0,0 +1,15 @@
+Package: dashBio
```

Comment:
@shammamah The package name is currently derived from the `project_shortname` as extracted from `package.json` by the 'transpiler': 

https://github.com/plotly/dash/blob/16e74e3232da8114f00b74036726b6ca0c650153/dash/development/_r_components_generation.py#L148

The `project_shortname` is parsed and converted to camelCase by this method, which works as long as the words are separated by `_` rather than `-`:

https://github.com/plotly/dash/blob/16e74e3232da8114f00b74036726b6ca0c650153/dash/development/_r_components_generation.py#L551-L553

I *think* this is desirable, but let me know if you have other thoughts. I'm grateful you asked because answering your question got me thinking about `dash-component-boilerplate`, which I will now need to edit slightly so less happens there before package 'transpilation' occurs 🙂 

@alexcjohnson

Reply:
@rpkyle Nice, I didn't see that! Was confused because on the Python side of things, generation of the package description is automated within our NPM build scripts.